### PR TITLE
Create `for Vim users.md`

### PR DESCRIPTION
**Pull request by chrisgrieser**
## Added
Overview Page for vim users. ([Context](https://discord.com/channels/686053708261228577/716028884885307432/989481680916082698))

## Checklist
- [x] before creating a new note, I searched the vault
- [x] new notes have the `.md` extension
- [x] (if applicable) attached images have descriptive file names
- [x] (if applicable) for new notes in the folder "04 - Guides, Workflows, & Courses", I added a link to them in one of the "for {group X}" overviews: https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/%F0%9F%97%82%EF%B8%8F+04+-+Guides%2C+Workflows%2C+%26+Courses

### CODE REVIEW

File: 04 - Guides, Workflows, & Courses/for Vim users.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+tags:
+- seedling
+---
+
+# for Vim users
+
+This site is for all information valuable to users familiar with vim, that want to use Obsidian's vim mode. 
+
+## Basics
+- You can enable Obsidian's vim mode under the settings under `Editor – Advanced – Vim key bindings`.
```
Comment (kometenstaub) [inline 04 - Guides, Workflows, & Courses/for Vim users.md:13]: Should it be `in the Settings under`?

File: 04 - Guides, Workflows, & Courses/for Vim users.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- Obsidian uses the [vim emulation from the CodeMirror Editor](https://github.com/codemirror/codemirror5/blob/master/keymap/vim.js), which does not include all vim commands, but most of them.
```
Comment (kometenstaub) [inline 04 - Guides, Workflows, & Courses/for Vim users.md:14]: I would link to the cm6 vim emulation repo. The `vim.js` deviated from the original cm5 version, and the repo also contains the adapter code.

File: 04 - Guides, Workflows, & Courses/for Vim users.md (hunk 1 above)
Comment (chrisgrieser) [reply 04 - Guides, Workflows, & Courses/for Vim users.md]: yes

File: 04 - Guides, Workflows, & Courses/for Vim users.md (hunk 2 above)
Comment (chrisgrieser) [reply 04 - Guides, Workflows, & Courses/for Vim users.md:14]: when you can tell me where the CM6 vim.js is? I couldn't find it in any of the CodeMirror repos

Comment (kometenstaub) [reply 04 - Guides, Workflows, & Courses/for Vim users.md:14]: https://github.com/replit/codemirror-vim

**Review by kometenstaub — APPROVED** [review]

**Pull request merged**